src/components/dashboard/clickable_cell_store.ts

```diff
@@ -11,16 +11,15 @@ import {
   invalidateEvaluationCommands,
 } from "../../types";
 
-type Garbage = ((position: CellPosition, env: SpreadsheetChildEnv) => void) | false;
```

Comment:
I'm really really sorry @rrahir to remove your `Garbage` type. I can imagine it took some effort to come up with a good name such as this one.
I can rename another type to `Garbage` if you want to keep it somewhere
:upside_down_face:

Reply:
https://media1.tenor.com/m/6cQdGuNHxugAAAAd/trash-throw-out.gif